src/pybind/mgr/dashboard/services/progress.py

```diff
@@ -14,37 +14,67 @@
 from .. import mgr, logger
 
 
+def _progress_event_to_dashboard_task_common(event, task):
+    if event['refs'] and isinstance(event['refs'], dict):
```

Comment:
Nit: I don't think `event['refs']` is necessary. If `refs` isn't in the dict, you'd get a `KeyError` and if it's not a dict, it is being checked by the second condition.

Reply:
I don't want to raise an exception if `refs` is not in the event dict. I just want to proceed gracefully.